bsp/esp-box-lite/idf_component.yml

```diff
@@ -9,10 +9,10 @@ tags:
   - bsp
 
 dependencies:
-  idf: ">=4.4.5"
+  idf: ">=4.4.5,<6.0"
 
   button:
-    version: ">=2.4"
+    version: "^2.4"
```

Comment:
Is it good to use this in `button` component? I think, better (safer) will be `~2.4`

Reply:
Button is already at v3.3.0 
![image](https://github.com/user-attachments/assets/db099a7a-7d31-4542-a23e-d1833da87244)

So I think that fixing to v2.* release is safe enough, because I don't expect any new 2.* release.

With `^2.4` we also support v2.5 which some users might need...

Reply:
Oh, ok.